operator/pkg/values/translation/translation_test.go

```diff
@@ -180,3 +181,382 @@ func TestGetTLSConfigValues(t *testing.T) {
 		})
 	}
 }
+
+func TestSetScannerComponentDisabled(t *testing.T) {
```

Comment:
Shouldn't this actually be something like
```suggestion
func TestSetScannerComponentPolicy(t *testing.T) {
```
?